staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -50,15 +49,14 @@ func WithAuthentication(handler http.Handler, auth authenticator.Request, failed
 		defer recordAuthMetrics(resp, ok, err, apiAuds, authenticationStart)
 		if err != nil || !ok {
 			if err != nil {
-				klog.Errorf("Unable to authenticate the request due to an error: %v", err)
+				klog.V(2).Infof("request (src %s: %s %s): unable to authenticate due to an error: %v", req.RemoteAddr, req.Method, req.URL.Path, err)
```

Comment:
user-agent?

Reply:
could be useful for source identification as well, added